aws/resource_aws_eks_node_group.go

```diff
@@ -246,6 +246,27 @@ func resourceAwsEksNodeGroup() *schema.Resource {
 					},
 				},
 			},
+			"update_config": {
+				Type:     schema.TypeList,
+				Required: false,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"max_unavailable": {
+							Type:          schema.TypeInt,
+							Required:      true,
+							ValidateFunc:  validation.IntBetween(1, 100),
```

Comment:
I think this should be IntBetween 1 & `max_size` of the node group

Reply:
According to the sdk documentation:

```
	// The maximum number of nodes unavailable at once during a version update.
	// Nodes will be updated in parallel. This value or maxUnavailablePercentage
	// is required to have a value.The maximum number is 100.
	MaxUnavailable *int64 `locationName:"maxUnavailable" min:"1" type:"integer"`
```

Reply:
Yes I see that now  👍